sources/packages/forms/src/form-definitions/institutionlocation.json

```diff
@@ -117,7 +117,7 @@
                     "name": "Disable inputs for Client not AEST",
                     "trigger": {
                         "type": "javascript",
-                        "javascript": "result = (data.clientType !== 'aest')"
+                        "javascript": "result = (data.clientType && data.clientType !== 'aest')"
```

Comment:
Why this change was introduced?

Reply:
To enable fields when the client Type was not present, this is used for Add location

Reply:
This expression can be simplified as 
` result =(data?.clientType !== 'aest'); `